packages/server/src/index.ts

```diff
@@ -136,35 +138,27 @@ function createServerAdapter<
     if (waitUntilPromises.length > 0) {
       await handleWaitUntils(waitUntilPromises);
     }
-    return response;
   }
 
-  function handleEvent(event: FetchEvent, ...ctx: Partial<TServerContext>[]) {
+  function handleEvent(event: FetchEvent, ...ctx: Partial<TServerContext>[]): void {
```

Comment:
FetchEvent listener should be `sync` and `void`. For example, CF Workers complain about that. (Integration tests are needed here)

Reply:
As with request listener, wanted to satisfy the types of `genericRequestHandler`.